### ISSUE TITLE
Link to Moonsharp

### ISSUE THREAD

**Issue by andybak** [fabSchneider/LiveLua#3]
Where you say:

> For this package to work add the moonsharp package in the Unity package manager from here: https://github.com/fabSchneider/moonsharp.git#v2.0.0-upm.533e8b0

Should that be:

https://github.com/fabSchneider/moonsharp/tree/v2.0.0-upm.533e8b0

?

**Comment by fabSchneider** [comment]
https://github.com/fabSchneider/moonsharp.git#v2.0.0-upm.533e8b0 is the link you need for the Unity package manager.

Check out their [manual](https://docs.unity3d.com/Manual/upm-git.html) on importing packages with specific revisions through the package manager. 

But it is a bit confusing, I agree. I will fix the readme to make it more clear.

**Comment by andybak** [comment]
Sorry - just realised that. In case anyone is as stupid as me, I'd suggest rewording that to:

> for this package to work add the moonsharp package via the Unity package manager using "Add from Git URL" with the following url: https://github.com/fabSchneider/moonsharp.git#v2.0.0-upm.533e8b0
